pkg/server/admin.go

```diff
@@ -2978,122 +2993,71 @@ func (s *adminServer) dataDistributionHelper(
 	var hasNext bool
 	for hasNext, err = it.Next(ctx); hasNext; hasNext, err = it.Next(ctx) {
 		row := it.Cur()
-		tableName := (*string)(row[0].(*tree.DString))
-		schemaName := (*string)(row[1].(*tree.DString))
-		fqTableName := fmt.Sprintf("%s.%s",
-			tree.NameStringP(schemaName), tree.NameStringP(tableName))
-		tableID := uint32(tree.MustBeDInt(row[2]))
-		dbName := (*string)(row[3].(*tree.DString))
-
-		// Look at whether it was dropped.
-		var droppedAtTime *time.Time
-		droppedAtDatum, ok := row[4].(*tree.DTimestamp)
-		if ok {
-			droppedAtTime = &droppedAtDatum.Time
-		}
+		tableID := uint32(*row[0].(*tree.DInt))
+		tableInfo, ok := tableInfosByTableID[tableID]
```

Comment:
Now that we aren't scanning Meta2, I think there may be a few cleanups we can do here:

1) The comment above at line 2991 is wrong.
2) This code is a little subtle now.  We have are using this to store tableInfo's, but the tableInfo that we actually put in the response via `dbInfo.TableInfo`, I believe, a different copy of the table Info we've put in this map, but, because the struct is holding a map, I think the replica update you do later on still works to update the map in dbInfo (although I haven't checked).  I wonder if there is a better way to arrange this (especially if you take my other cleanup suggestion below)